__tests__/integration.js

```diff
@@ -82,6 +84,29 @@ test('--mutex network', async () => {
   ]);
 });
 
+test('--mutex network 2', async () => {
+  const cwd = await makeTemp();
+  await fs.writeFile(path.join(cwd, '.yarnrc'), '--mutex network\n');
```

Comment:
You should probably use a random port for each run to make sure we don't clash with any parallel tests.